<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Publications</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Home</div>
<div class="menu-item"><a href="main.html" class="current">Main</a></div>
  <div class="menu-category">About</div>
<div class="menu-item"><a href="education.html">Academic Experience</a></div>
<div class="menu-item"><a href="resume.pdf">Resume</a></div>
<div class="menu-category">Research</div>
<div class="menu-item"><a href="researchint.html">Research Interests</a></div>
<div class="menu-item"><a href="publications.html">Publications</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Publications</h1>
</div>
<h2>Book chapters</h2>
<li>
 <strong><u>S. Das</u></strong>, H. Vikalo, and A. Hassibi,
 <a href="http://www.wiley.com/WileyCDA/WileyTitle/productCd-0470641908.html">
<b>"Affinity-Based Biosensors: Stochastic Modeling and Figures of Merit,"</b></a>
in <em>CMOS Biomicrosystems: Where Electronics Meet Biology</em>, Wiley 2011.<br><br>
</li>
  
<h2>Publications</h2>
  
<li>S.Barik, <strong><u>S. Das</u></strong> and H. Vikalo, <a href="http://www.biorxiv.org/content/early/2016/12/26/096768">&ldquo; Viral Quasispecies Reconstruction via Correlation Clustering. &rdquo; </a><i>bioRxiv</i><br><br>
 </li>
  
<li><strong><u>S. Das</u></strong> and H. Vikalo,<a href="http://ieeexplore.ieee.org/abstract/document/7784705/">&ldquo; Optimal Haplotype Assembly via a Branch-and-Bound Algorithm. &rdquo;</a> <i>IEEE Transactions on Molecular, Biological and Multiscale Communications</i> , vol 3, issue 1, pp 1-12, 2017<br><br>

  <p>Preliminary version in <strong><u>S. Das</u></strong> and H. Vikalo, <a href="http://ieeexplore.ieee.org/abstract/document/7032339/">&ldquo; Optimal Haplotype Assembly with Statistical Pruning. &rdquo;</a> <i>IEEE International Workshop on Genomic Signal Processing and Statistics (GENSIPS)</i>, Atlanta, GA, December 2014.</p><br><br>

<li><strong><u>S. Das</u></strong> and H. Vikalo, <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1408-5">&ldquo; SDhaP: Haplotype Assembly for Diploids and Polyploids via Semi-Definite Programming,&rdquo; </a> <i>BMC Genomics</i>, vol 16, pp 260, 2015.<br><br>

  <p>Preliminary version in <strong><u>S. Das</u></strong> and H. Vikalo, "Single Individual Haplotyping with Low Rank Semidefinite Programming." <i>NIPS Workshop on Machine Learning and Computational Biology</i>, Montreal, CA, December 2014.</p><br><br>

<li><strong><u>S. Das</u></strong> and H. Vikalo, <a href="http://www.biomedcentral.com/1471-2105/14/129/">&ldquo;Base calling for high-throughput short-read sequencing: Dynamic programming solutions,&rdquo;</a> <i>BMC Bioinformatics</i>, vol. 60, no. 10, pp. 6576 - 6601. 2013.<br><br>

  <p>Preliminary version in <strong><u>S. Das</u></strong> and H. Vikalo, <a href="http://ieeexplore.ieee.org/xpls/abs_all.jsp?arnumber=6120377&tag=1"> &ldquo; Base-calling for Illumina's next-generation sequencing via Viterbi algorithm.</a> &rdquo; <i>49th Annual Allerton Conference on Communication, Control, and Computing</i>, Monticello, IL, September 2011, pp. 1733-1736 (invited).</p><br><br>

<li><strong><u>S. Das</u></strong> and H. Vikalo, <a href="http://bioinformatics.oxfordjournals.org/content/28/13/1677.short">&ldquo;OnlineCall: Fast online parameter estimation and base calling for Illumina's next generation sequencing,&rdquo;</a> <i> Bioinformatics</i>, vol. 28, no. 13, pp. 1677-1683, 2012.<br><br>

  <p>Preliminary version in <strong><u>S. Das</u></strong> and H. Vikalo, <a href="http://ieeexplore.ieee.org/xpls/abs_all.jsp?arnumber=5719675"> &ldquo; Model-based sequential base calling for Illumina sequencing.</a> &rdquo; <i>IEEE International Workshop on Genomic Signal Processing and Statistics (GENSIPS)</i>, Cold Spring, NY, November 2010.</p><br><br>

<li><strong><u>S. Das</u></strong>, H. Vikalo and A. Hassibi <a href="http://scitation.aip.org/content/aip/journal/jap/105/10/10.1063/1.3116125">&ldquo;On scaling laws of biosensors: a stochastic approach,&rdquo;</a> <i>Journal of Applied Physics</i>,  vol. 105, no. 10, May 2009, pp. 102021-7.<br><br>

  <p>Preliminary version in <strong><u>S. Das</u></strong>, H. Vikalo and A. Hassibi <a href="http://ieeexplore.ieee.org/xpls/abs_all.jsp?arnumber=5174363"> &ldquo; Stochastic modeling of reaction kinetics in biosensors using the Fokker-Planck equation.</a> &rdquo; <i>IEEE International Workshop on Genomic Signal Processing and Statistics (GENSIPS)</i>, Minneapolis, MN, 2009.</p><br><br>

<h2>Unpublished Manuscripts</h2>
<li> <strong><u>S. Das</u></strong>, M. Shamaiah and S. Moharir, &ldquo; <a href="gene_networks.pdf">Inferring gene regulatory network structure.</a> &rdquo;


<div id="footer">
<div id="footer-text">
Page generated 2015-01-11 23:29:31 CST, by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</div>
</body>
</html>
